strawberry_django/fields/base.py

```diff
@@ -43,11 +60,86 @@ def is_basic_field(self) -> bool:
         """
         return False
 
-    @property
-    def type(  # noqa: A003
+    @cached_property
+    def django_type(self) -> type[utils.WithStrawberryDjangoObjectDefinition] | None:
```

Comment:
nit: could we do this?

```suggestion
    def django_type(self) -> type[WithStrawberryDjangoObjectDefinition] | None:
```

Reply:
Done! :)